src/test/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolderTest.java

```diff
@@ -221,6 +221,14 @@ public void testIsSuppressedAfterEventStart() throws Exception {
         assertTrue(SuppressWarningsHolder.isSuppressed(event), "Event is not suppressed");
     }
 
+    @Test
+    public void testIsSuppressedAfterEventStart2() throws Exception {
+        populateHolder("check", 100, 100, 350, 350);
+        final AuditEvent event = createAuditEvent("check", 100, 0);
```

Comment:
even I completely against of testing like this .....
looks like this module is so forgotten by us ....  because we try to deprecate it eventually.
This module is hack in checkstyle.

I am ok with such fix